pkgs/by-name/td/tdb/package.nix

```
@@ -51,10 +69,21 @@ stdenv.mkDerivation rec {
       "--bundled-libraries=NONE"
       "--builtin-libraries=replace"
     ]
-    ++ lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) [
-      "--cross-compile"
-      "--cross-execute=${stdenv.hostPlatform.emulator buildPackages}"
-    ];
+    ++ lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) (
+      [
+        "--cross-compile"
+      ]
+      ++ (
+        if (stdenv.hostPlatform.emulatorAvailable buildPackages) then
+          [
+            "--cross-execute=${stdenv.hostPlatform.emulator buildPackages}"
+          ]
+        else
+          [
+            "--cross-answers=answers"
+          ]
+      )
+    );
```

```suggestion
      (if (stdenv.hostPlatform.emulatorAvailable buildPackages) then
          "--cross-execute=${stdenv.hostPlatform.emulator buildPackages}"
        else
          "--cross-answers=answers")
      ]
    );
```
I supect this `if` expression will stay one flag on each branch, so we could make it a bit terser this way.